cypress/support/api/apiTests.spec.js

```diff
@@ -0,0 +1,77 @@
+import {DELETE, GET, POST, PUT} from "cypress/support/api/apiUtils.js";
+import {
+    createDashboard,
+    deleteDashboardById,
+    getActivityItemId,
+    updateDashboardById
+} from "cypress/support/api/endpoints.js";
+
+describe('CRUD Tests', () => {
+    it('should retrieve activity item by ID', () => {
+
+        const projectName = 'DEMO DASHBOARD';
+        const itemId = 10;
+
+        // Make the GET request using the utility function
+        GET(getActivityItemId(projectName, itemId)).then(response => {
```

Comment:
I'd be more logical to replace this request to active item on a getDashboardById (`/v1/${projectName}/dashboard`), because your - get active item by id is more related to items rather than to dashboards.